src/frontend/components/TimedTextCreationForm/index.tsx

```diff
@@ -92,7 +90,7 @@ export const TimedTextCreationForm = ({
     choices.filter((language) => !excludedLanguages.includes(language.value));
 
   const onSelectChange = (
-    option: ValueType<SelectOption, false>,
+    option: Nullable<SelectOption>,
     { action }: ActionMeta<SelectOption>,
   ) => {
     if (action === 'select-option' && option && isSelectOption(option)) {
```

Comment:
According to `onSelectChange` signature, you can remove `isSelectOption` type guard and update your if statement like this:
```ts
if (action === 'select-option' && option) { ...
```

Reply:
You are right :+1: